doc/api/deprecations.md

```diff
@@ -2518,6 +2518,18 @@ Type: Documentation-only
 Prefer [`response.socket`][] over [`response.connection`] and
 [`request.socket`][] over [`request.connection`].
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: process._tickCallback
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/
```

Comment:
Should this be the url of this PR?   or is it another one from a different part of the process?

Reply:
It'll be the URL of this PR

Reply:
ok,  thanks.  i'll update that too